lib/gherkin_format.rb

```diff
@@ -22,10 +22,19 @@ def format(file, options = {})
     output = format_string input, file
     return if input == output
 
-    File.write(file, output) if options.key? :replace
+    if options.key? :replace
+      File.write(file, output)
+      puts "File #{file} was not formatted well. Fixed it for you!"
+    else
+      puts "File #{file} is not formatted well. Re-run with --replace"
+    end
+
+    fail_with 'Terminating. Some files may not have been checked.'
+  end
 
-    puts "File #{file} is not formatted well."
-    raise "File #{file} is not formatted well."
+  def fail_with(message)
+    puts message
+    exit 1
```

Comment:
raising in inner methods is a better way as just `exit`. Try to stick with `raise` and just catch the exception on a higher level.

Reply:
done